src/client/components/HomePage/LandingGraphicSliver.jsx

```diff
@@ -103,9 +103,9 @@ const useStyles = makeStyles((theme) =>
 const LandingGraphicSliver = () => {
   const classes = useStyles()
   const topPaddingMultipler = () => {
-    const theme = useTheme()
-    const isMediumWidth = useMediaQuery(theme.breakpoints.up('md'))
-    const isDesktopWidth = useMediaQuery(theme.breakpoints.up('lg'))
+    const theme = UseTheme()
+    const isMediumWidth = UseMediaQuery(theme.breakpoints.up('md'))
+    const isDesktopWidth = UseMediaQuery(theme.breakpoints.up('lg'))
```

Comment:
`useTheme` and `useMediaQuery` really are hooks and should hence retain camelCase naming. However, `topPaddingMultipler()` is also meant to be a hook, and should hence follow hook convention.

I would like you to reinstate the camelCasing for `useTheme` and `useMediaQuery`, and rename `topMaddingMultipler()` to `useTopPaddingMultiplier()` (note the typo)